pycfmodel/resolver.py

```diff
@@ -19,7 +20,7 @@ def _extended_bool(value) -> bool:
 
 
 class _BooleanModel(BaseModel):
-    bool_value: bool
+    bool_value: SemiStrictBool
```

Comment:
this is not strictly necessary, it can reveted

Reply:
Check [15863aa](https://github.com/Skyscanner/pycfmodel/pull/91/commits/15863aa20e7d1f1d34f48229462ddc3275fbafbe) since got it reverted and test pass.